src/pages/FeedPage/FeedPage.tsx

```diff
@@ -1,10 +1,8 @@
+import React from 'react'
 
-
-const FeedPage = () => {
-
-    return (
-        <div>This is the feed page!</div>
-    );
+const FeedPage: React.FC<unknown> = () => {
```

Comment:
For these can we do JSX.Element? [These docs](https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/function_components/) indicate that React.FC is discouraged